### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# presto-jdbc-java6
+
+Presto JDBC driver compatible with Java 6
+
+# Motivation 
+
+This is still common to see companies dependin on Java 6 VMs for old systems.
```
Comment (pnowojski) [inline README.md]: dependinG

Comment (pnowojski) [inline README.md]: Many companies are still using Java 6 VM's for (...) ?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# Building
+
+## Setup JDK 6 toolchain
+
+Build script is `mvn` based and depends on [`toolchains`](https://maven.apache.org/guides/mini/guide-using-toolchains.html) feature of Maven.
+
+We need that as new version of maven require JRE 7 to run itself.
+And we want to compile project with JDK 6.
+
+To configure Java 8 toolchain in `~/.m2/toolchains.xml` put
```
Comment (pnowojski) [inline README.md]: Java 6?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        </configuration>
+    </toolchain>
+</toolchains>
+```
+
+Replace contents of `jdkHome` with path matching your machine.
+
+## Build
+
+To just build execute:
```
Comment (pnowojski) [inline README.md]: To build just execute?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+```
+mvn install -DskipTests=true  
+```
+
+## Tests
+
+To run integration tests you need to explicitly specify path to JDK 8, will be used to spawn mock Presto instance.
```
Comment (pnowojski) [inline README.md]: , which will be used ?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+```
+ mvn test -Djava8.home=/usr/lib/jvm/java-8-oracle
+```
+
+Replace value of `java8.home` property with path valid on your system.
+
+## Artifacts
+
+Build produces single library artifact `presto-jdbc-java6-<version>.jar`.
```
Comment (pnowojski) [inline README.md]: Build produces single library artifact ... without any external dependencies, which will be available in .... ?

Comment (losipiuk) [inline README.md]: I like mine more

**Pull request closed without merging**